variables.tf

```diff
@@ -1,35 +1,35 @@
-variable "server_name" {
+variable "name" {
   type        = string
   description = <<-EOT
     The name to give the server, this will form the local hostname and appear in the AWS ec2 console.
     The name should be unique within your region.
     The name will be used as the value for the "Name" tag on the server.
     The name should give some indication of the purpose of the server.
-    This value is ignored when overriding with the server_id.
-    If both the server_name and server_id are empty, no server will be created or selected and data will be empty.
+    This value is ignored when overriding with the id.
+    If both the name and id are empty, no server will be created or selected and data will be empty.
   EOT
   default     = ""
 }
 
-variable "server_owner" {
+variable "owner" {
   type        = string
   description = <<-EOT
     The value of the "Owner" tag which will be added to resources, usually this is your email address.
     Adding your email address as a tag on resources allows your teammates to contact you about the resource.
     This can help prevent accidental deletion of resources or unexpected charges.
-    This value is ignored when overriding with the server_id.
+    This value is ignored when overriding with the id.
   EOT
   default     = ""
 }
 
-variable "server_type" {
+variable "type" {
```

Comment:
I understand overriding the name and ID fields but not the type of the server with respect to the EC2 instance type. Can you elaborate on this variable being overriden by id?

Reply:
Was this supposed to be image_id overrides it?

Reply:
This is not the ec2 type, it is a custom pre-chosen type that the module provides. The idea is to reduce the cost of choice paralysis by providing a smaller subset of generally acceptable "types". Those types are detailed in the server module and the description of this variable:
https://github.com/rancher/terraform-aws-server/blob/main/modules/server/types.tf

Reply:
I must have removed the details about this from the readme at some point, I will add it back in.

Reply:
done